### DISCUSSION TITLE
Need help in dataset .txt file

### DISCUSSION THREAD

**Discussion by ahthserhsluk** [RizwanMunawar/yolov8-object-tracking#3] [General]
hey bro

while creating the label file that has **<object-class-id> <x> <y> <width> <height>
**
do we have to make one txt file with one image
or one file with all image data?

**Comment by RizwanMunawar** [comment]
@ahthserhsluk one file for complete data.
For more details, you can take a look at link mentioned below.
https://medium.com/augmented-startups/train-yolov8-on-custom-data-6d28cd348262
